docs/permissions-and-roles.md

```diff
@@ -0,0 +1,31 @@
+# Permissions and roles
+
+'mediator':
+
+* For users who frequently create dynamic content on the website.
+* Can create and edit events, articles, pages, and campaigns in
+  /admin/content.
+
+'editor':
+
+* For users who, in addition to content creation, are responsible for
+  static content (e.g., opening hours) and the site's structure.
+* Can coordinate and manage the work of mediators.
+* Has access to the same as mediator plus all content types in
+  /admin/content, as well as taxonomy and menus in /admin/structure.
+
+'local_administrator':
+
+* For users who need to configure the website in terms of settings,
+  appearance, integrations, general information, library usage, etc.
+* Has access to the same as editor plus the site's configuration
+  options in /admin/config, user creation in /admin/people, and
+  changing the site's appearance in /admin/appearance.
+* This is the role assigned to trusted users on regular libraries.
```

Comment:
```suggestion
* This is the role assigned to trusted users on Editor libraries.
```

Please consider referring to [the name of the plan used by the business and elsewhere in the docs](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/5bb77ed095e4690871a51fd65694023d62a17a61/docs/configuration-management.md#L14-L14). 

The name overlaps with the user role but in practice I do not think this has been a problem.